### ISSUE TITLE
Иногда на точках спавна игрока появляются "призрачные" игроки.

### ISSUE THREAD

**Issue by ghost** [FWGS/xash3d-fwgs#1157]
Было замечено, что иногда на точках спавна игрока появляются "призрачные" игроки.
"Призрачность" заключается в том, что в качестве ника у них стоит "******", они не двигаются и иногда то либо не видны, либо видны как модель HEV'ика, их так-же можно убить и можно в них застрять, что доставляет неудобства.
Пытался кикнуть такого "призрачного" игрока через RCON от сервера, а он оказывается даже в status сервера не отображается.
Есть предположение, что такие "призрачные" игроки в основном появляются, если кто-то делает слишком много переподключений к серверу.

**Comment by CortexReaver** [comment]
Yes, I can confirm such behavior. Seen it myself in a multiplayer game.

**Comment by a1batross** [comment]
New/old protocol?

пт, 9 дек. 2022 г., 20:30 CortexReaver ***@***.***>:

**Comment by CortexReaver** [comment]
> New/old protocol?
I don't know, It was some build from mid-late 2022, not so long ago.

**Comment by a1batross** [comment]
So I think this engine branch is affected by this bug too.

**Comment by a1batross** [comment]
Three years later, I never encountered this bug online. Closing.

Create a new bugreport using latest engine build if that's still an issue.
